config/routes.rb

```diff
@@ -209,6 +209,10 @@
           resources :materials, path: 'files'
         end
       end
+
+      resources :course_users, only: [] do
+        resources :experience_points_records, only: [:new, :create, :index]
```

Comment:
Do you mean that I should also namespace the controller under `Course::Admin::`? That will make it a 'setting' with view headers and all right? I think viewing and awarding EXP should not be considered a setting for EXP.

Also, admins might not be the only users for this: we also want to make the a student's EXP history viewable by the student.

Is it better to leave it as it is?